integrations/github/jwt.go

```diff
@@ -46,13 +46,13 @@ func (i integration) NewClient(ctx context.Context) (*github.Client, error) {
 	if pat := data.Get(vars.PAT); pat.IsValid() {
 		return github.NewTokenClient(ctx, pat.Value()), nil
 	} else {
-		return i.newClientWithInstallJWT(data)
+		return newClientWithInstallJWT(data)
 	}
 }
 
-// NewClientWithInstallJWT returns a GitHub client that
+// newClientWithInstallJWT returns a GitHub client that
 // uses a newly-generated GitHub app installation JWT.
-func (i integration) newClientWithInstallJWT(data sdktypes.Vars) (*github.Client, error) {
```

Comment:
what's the motivation behind making it a standalone function?

Reply:
Even though I don't use this specific function in this PR, it doesn't use any of the integration structs's fields, so there's no point in keeping it attached to the integration.